catalog/solr/catalog-solr-core/src/main/java/ddf/catalog/source/solr/SolrCatalogProviderImpl.java

```diff
@@ -363,7 +363,7 @@ public DeleteResponse delete(DeleteRequest deleteRequest) throws IngestException
       deleteListOfMetacards(deletedMetacards, identifierPaged, attributeName);
     }
 
-    return new DeleteResponseImpl(deleteRequest, null, deletedMetacards);
+    return new DeleteResponseImpl(deleteRequest, deleteRequest.getProperties(), deletedMetacards);
```

Comment:
I could be misremembering, but it's probably worth checking that there's nothing sensitive inside the request properties that we don't want to send in the response.

Reply:
I would agree. I think we should leave the request properties where they are and check them if there is a property we're looking for (I just made an update to do that to check for the skip_versioning property https://github.com/codice/ddf/pull/6789).

Reply:
Yea makes sense. I'll close this one and make an update to yours

Reply:
Okay yea I see your change now. I'll drop that change in my PR and we can have these tested together.